recipes/torchfsm/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "torchfsm" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/qiauil/torchfsm/releases/download/v{{ version }}/torchfsm-{{ version }}.tar.gz
+  sha256: bd0ae6e48c5daca1e83d7f62530a916e23d93131959804a3c84e963ef2cf9e18
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - pip
+    - setuptools
+  run:
+    - python >={{ python_min }}
+    - ipython
+    - matplotlib-base
+    - numpy
+    - pytorch
+    - tqdm
+    - vape4d
+
+test:
+  imports:
+    - torchfsm
+  # commands:
+  #   - pip check
```

Comment:
Why? We prefer to fix these instead of skipping. Otherwise we force all packages downstream to also skip.